clade/proto/sync.proto

```diff
@@ -4,6 +4,22 @@ package clade.sync;
 
 import "schema.proto";
 
+enum ColumnRole {
+  // Denotes a column that that has the old value for a primary key column
+  OLD_PK = 0;
+  // Denotes a column that that has the new value for a primary key column
+  NEW_PK = 1;
+  // Denotes whether a particular column has changed in value
+  CHANGED = 2;
```

Comment:
I am still not personally confident about how TOAST columns actually work. We may need to change more here or not. But nbd for now, just mentioning.